src/activities/Elsa.Activities.Http/Extensions/RouteTableExtensions.cs

```diff
@@ -15,13 +15,15 @@ public static class RouteTableExtensions
     
     public static void AddRoutes(this IRouteTable routeTable, IEnumerable<Trigger> triggers)
     {
-        var paths = Filter(triggers).Select(Deserialize).Select(x => x.Path).ToList();
+        var paths = Filter(triggers).Select(Deserialize).Select(x => x.Path)
+            .Where(x => !x.Contains("//")).ToList();
         routeTable.AddRange(paths);
     }
 
     public static void AddRoutes(this IRouteTable routeTable, IEnumerable<Bookmark> bookmarks)
     {
-        var paths = Filter(bookmarks).Select(Deserialize).Select(x => x.Path).ToList();
+        var paths = Filter(bookmarks).Select(Deserialize).Select(x => x.Path)
+            .Where(x => !x.Contains("//")).ToList();
         routeTable.AddRange(paths);
```

Comment:
Although this will certainly work, it might be even better to do the filtering at the lowest level possible, which is inside the `Add` method of the `RouteTable` class itself - that way, all code paths leading into there will have this filtering logic applied. For example:

```csharp
public void Add(string path)
{
    if (path.Contains("//"))
    {
        _logger.LogWarning("Path cannot contain double slashes. Ignoring path: {Path}", path);
        return;
    }

    Routes.TryAdd(path, path);
}
```

Reply:
I see. I falsely assumed that the (I)RouteTable was coming from ASP.NET Core itself. But since it is part of else, it makes sense to do it there.